lstchain/reco/dl1_to_dl2.py

```diff
@@ -102,12 +102,8 @@ def train_energy(train,
     The trained model
     """
```

Comment:
What if a `config_file` is passed but the loading of the config fails?
What is the interest of removing this try/except?

Reply:
Sorry I didn't see the comment in your commit.
I see your point now.

Reply:
So I think we can resolve this conversation?

Reply:
Yes 👍